lib/fiber/intrinsic-jsx.ts

```diff
@@ -1,53 +1,60 @@
 import type * as Props from "@tscircuit/props"
 
-declare global {
+export interface TScircuitElements {
```

Comment:
Casing, so we use google style single word casing and tscircuit we generally consider a single word
```suggestion
export interface TscircuitElements {
```